tests/functional/extension_functions/test_approximation_functions.py

```diff
@@ -0,0 +1,126 @@
+from collections.abc import Iterable
+from typing import Callable
+
+import duckdb
+import pytest
+from ibis.expr.types.relations import Table
+
+from tests.consumers.acero_consumer import AceroConsumer
+from tests.consumers.duckdb_consumer import DuckDBConsumer
```

Comment:
Nit:
if you import the `AceroConsumer` and `DuckDBConsumer` into the `tests/consumers/__init__.py` file, then these two lines would collapse into `from tests.consumers import AceroConsumer, DuckDBConsumer` -- and I think the benefits of that would continue as more consumers are added.